joint_trajectory_controller/include/joint_trajectory_controller/joint_trajectory_controller_impl.h

```diff
@@ -811,6 +818,23 @@ setHoldPosition(const ros::Time& time, RealtimeGoalHandlePtr gh)
   curr_trajectory_box_.set(hold_trajectory_ptr_);
 }
 
+template <class SegmentImpl, class HardwareInterface>
+inline bool JointTrajectoryController<SegmentImpl, HardwareInterface>::
+checkStates(const std::vector<double>& /* old_desired_position */,
```

Comment:
Do you have a reason for considering only position checks? (besides that it fits our needs)
If not, I would pass the whole states, not only the position.

Reply:
Done.